src/backend/joanie/core/models/products.py

```diff
@@ -1236,13 +1236,16 @@ def get_count_installments_paid(self):
             for installment in self.payment_schedule
         )
 
-    def get_position_last_paid_installment(self):
-        """Get the position of the last installment paid in the payment schedule."""
-        last_paid_index = None
+    def get_position_of_last_installment(self, state):
```

Comment:
IMO `index` is better than `position` here
```suggestion
    def get_index_of_last_installment(self, state):
```